src/main/java/de/adesso/anki/battle/world/Body.java

```diff
@@ -1,14 +1,20 @@
 package de.adesso.anki.battle.world;
 
+import java.util.List;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)